### DISCUSSION TITLE
Add the estimated amount of damage next to the exp drop.

### DISCUSSION THREAD

**Discussion by Milonny** [runelite/runelite#15205] [Q&A] [answered]
ass the title says: Add the estimated amount of damage next to the exp drop.
for example a drop with 4 range 1 hitpoints xp would have a (1) next to it

**Comment by Enriath** [comment, marked as answer]
This is already possible with a hub plugin: [InstantDamageCalculator](https://github.com/geeckon/instant-damage-calculator)

**Reply by Milonny** [reply]
oh my god thank you!
